lib/active_merchant/billing/gateways/mit.rb

```diff
@@ -0,0 +1,244 @@
+          amount: amount(money)
+        }
+        post[:key_session] = self.options[:key_session]
+
+        post_to_json = post.to_json
+        puts(post_to_json)
+        post_to_json_encrypt = encrypt(post_to_json, self.options[:key_session])
+
+        final_post = post_to_json_encrypt + '-' + self.options[:user]
+        commit('capture', final_post)
+      end
+
+      def refund(money, authorization, options = {})
+        puts('---------------------------------------------')
+        puts('Refund')
+        puts('---------------------------------------------')
+        post = {
+          operation: 'Refund',
+          id_comercio: self.options[:id_comercio],
+          user: self.options[:user],
+          apikey: self.options[:apikey],
+          transaccion_id: options[:transaccion_id],
+          auth: authorization,
+          amount: amount(money)
+        }
+        post[:key_session] = self.options[:key_session]
+
+        post_to_json = post.to_json
+        puts(post_to_json)
+        post_to_json_encrypt = encrypt(post_to_json, self.options[:key_session])
+
+        final_post = post_to_json_encrypt + '-' + self.options[:user]
+        commit('refund', final_post)
+      end
+
+      # Currently unsupported
+      # def void(authorization, options={})
+      #   puts('---------------------------------------------')
+      #   puts('Void')
+      #   puts('---------------------------------------------')
+      #   refund(options["money"], authorization, options)
+      # end
+
+      def verify(credit_card, options = {})
+        MultiResponse.run(:use_first_response) do |r|
+          r.process { authorize(100, credit_card, options) }
+          r.process(:ignore_result) { void(r.authorization, options) }
+        end
+      end
+
+      def supports_scrubbing?
+        false
+      end
+
+      private
+
+      def add_customer_data(post, options)
+        post[:email] = options[:email] || 'nadie@mit.test'
+      end
+
+      def add_address(post, creditcard, options); end
+
+      def add_invoice(post, money, options)
+        post[:amount] = amount(money)
+        post[:currency] = (options[:currency] || currency(money))
+        post[:reference] = options[:order_id]
+        post[:transaccion_id] = options[:order_id]
+      end
+
+      def add_payment(post, payment)
+        post[:installments] = 1
+        post[:card] = payment.number
+        post[:expmonth] = payment.month
+        post[:expyear] = payment.year
+        post[:cvv] = payment.verification_value
+        post[:name_client] = [payment.first_name, payment.last_name].join(' ')
+      end
+
+      def parse(body)
+        {}
+      end
+
+      def commit(action, parameters)
+        url = (test? ? test_url : live_url)
+        raw_response = ssl_post(url, parameters, { 'Content-type' => 'text/plain' })
+        puts(raw_response)
+        response = JSON.parse(decrypt(raw_response, self.options[:key_session]))
+        puts(response)
```

Comment:
If you'd like to view the request and response data in your testing, feel free to add `DEBUG_ACTIVE_MERCHANT=true` to your test command.